### ISSUE TITLE
Failure to refresh upon switching project

### ISSUE THREAD

**Issue by FPtje** [timbrel/GitSavvy#1363]
Quite a few times now I've found myself staring at an out-of-date STATUS or BRANCHES tab, being thoroughly confused about what's going on, e.g. why certain files don't exist, or why the STATUS screen shows nothing modified even though I know for sure there are some modified files. The confusion usually lasts a couple of seconds before I realize I need to close the tab and re-open it to refresh it.

## How to reproduce

Make sure you have two projects, let's call them A and B. Project A has one directory tracked by git. 

- Open Project A (Ctrl + Alt + P)
- Make sure it's on master and has no modified files
- Open a STATUS and a BRANCHES view in their own tabs
- Switch to project B using Ctrl + Alt + P
- Switch to a different branch in project A through your terminal
- Edit some file from project A through some other means (`nano`, `vim`, `echo/cat`, whatever floats your boat)
- Switch back to project A using Ctrl + Alt + P
- Observe in BRANCHES tab that it looks like you're still on master
- Observe in STATUS tab that your manually edited file does not show up as modified
- Close both tabs and re-open them
- Observe changes

## Probable cause

The tabs refresh perfectly fine upon Sublime startup. That implies that there is some logic to refresh on certain events. This logic doesn't appear to be fired when a project is switched to. That probably needs to be added.

**Comment by kaste** [comment]
A first quick reply: Is this a rather new bug or like forever so? (We *should* refresh on focus, so for example `ctrl+tab+tab` should trigger a refresh. Or mouse clicking here and there.)  Do you use GitSavvy *project* settings here?

**Comment by FPtje** [comment]
I've noticed in in the last month or so, I'm not sure if this was a bug before that because I didn't use projects this heavily in the past. Using `Ctrl+Tab` > `Tab` to refresh would probably work, but it's a similar workaround as closing the tab and re-opening it, or restarting Sublime.

What do you mean by GitSavvy project settings? These are my GitSavvy user settings:

```json
{
  "git_path": "git",
  "show_commit_diff": true,
  "inline_diff_auto_scroll": true,
  "silent_fallback": true,
  "load_additional_codecs": true,
  "fallback_encoding": "gbk"
}
```

My project settings (`Project` > `Edit Project`) have no mention of GitSavvy.

**Comment by kaste** [comment]
Hm, this has been introduced by f29a0c9, published as v2.25.3 (Aug 10 2020).

**Comment by FPtje** [comment]
Nice find!
